tensorflow/lite/micro/kernels/all_ops_resolver.cc

```diff
@@ -70,6 +70,7 @@ AllOpsResolver::AllOpsResolver() {
   AddBuiltin(BuiltinOperator_RELU, Register_RELU());
   AddBuiltin(BuiltinOperator_RELU6, Register_RELU6());
   AddBuiltin(BuiltinOperator_MEAN, Register_MEAN());
+  AddBuiltin(BuiltinOperator_EXPAND_DIMS, Register_EXPAND_DIMS());
```

Comment:
`ld.lld: error: undefined symbol: tflite::ops::micro::Register_EXPAND_DIMS()
>>> referenced by all_ops_resolver.cc:73 (third_party/tensorflow/lite/micro/kernels/all_ops_resolver.cc:73)
>>>               blaze-out/k8-asan-fastbuild/bin/third_party/tensorflow/lite/micro/kernels/_objs/all_ops_resolver/all_ops_resolver.pic.o:(tflite::ops::micro::AllOpsResolver::AllOpsResolver())
clang: error: linker command failed with exit code 1 (use -v to see invocation)`

Reply:
Hi @rthadur . Sorry for the late reply. Could you please share with me how you produced this error, like the build command you used, the toolchain (which version of clang), OS etc, as I wasn't able to re-produce this linker error? Thanks!

Reply:
@petewarden can you please assist with this ?